librz/core/canalysis.c

```diff
@@ -378,9 +378,25 @@ static void bb_info_print(RzCore *core, RzAnalysisFunction *fcn, RzAnalysisBlock
 		inputs += (bb2->jump == bb->addr) + (bb2->fail == bb->addr);
 	}
 	if (bb->switch_op) {
-		RzList *unique_cases = rz_list_uniq(bb->switch_op->cases, casecmp);
-		outputs += rz_list_length(unique_cases);
-		rz_list_free(unique_cases);
+		RzAnalysisCaseOp *case_op;
+		void **it;
+		RzPVector *unique_cases = rz_pvector_new(NULL);
+		rz_pvector_foreach (bb->switch_op->cases, it) {
+			case_op = *it;
+			bool found = false;
+			void **it_unique;
+			rz_pvector_foreach (unique_cases, it_unique) {
+				if (casecmp(*it_unique, case_op) == 0) {
+					found = true;
+					break;
+				}
+			}
+			if (!found) {
+				rz_pvector_push(unique_cases, case_op);
+			}
+		}
+		outputs += rz_pvector_len(unique_cases);
+		rz_pvector_free(unique_cases);
```

Comment:
since this is only adding some nums, instead of wasting memory and cpu cycles, just increment `outputs` by `1`.